kubernetes_state/test_kubernetes_state.py

```diff
@@ -63,48 +63,18 @@ class TestKubernetesState(AgentCheckTest):
         NAMESPACE + '.container.waiting',
     ]
 
-    def test__get_kube_state(self):
```

Comment:
I removed these two tests as we are based on PrometheusCheck that already is tested